h/static/scripts/directive/test/privacy-test.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var PrivacyController = require('../privacy').PrivacyController;
+var PrivacyController = require('../privacy')._Controller;
```

Comment:
I guess you're changing this in order to be consistent with the changes you've made to the `groupList` directive. That's fine, but ideally we'd keep changes like this in separate commits, if not in an entirely separate PR.

But I'm not a great fan of the change to start with, so perhaps just back this out?